_pages/en/about-us.html

```diff
@@ -5,7 +5,23 @@
 lang: en
 permalink: /about-us.html
 ---
+<div class="team-intro">
+<p>This website is a platform where we, IT Strategy, are sharing our plan for becoming a digital government, our progress, and all the tools, information and other resources needed to support this transformation across IITB and ESDC as a whole. 
+It is a place that gives us an opportunity to express our professional opinion and ideas.</p>
 
+<p>The ESDC IT Strategy team is working in the open and welcomes any feedback while work is still in progress.
```

Comment:
Could this whole section be shorter and punchier?  
Eg. We are the ESDC IT Strategy team. We are passionate about working in the open and strongly believe the GC is better when work is visible and can be reused across groups, branches and departments. We also strongly advocate for the use of open source code and DevOps.
We manage our work in an agile way in 2 week sprints using a kanban board in GitHub and have standups, sprint planning sessions and regular retrospectives.

*I'm not really big on putting in a line about us being 'a fun team' because if feels less true if we have to say it. But if we were to put a line in like that maybe something like "we've been known to end our planning sessions playing online games'.

Reply:
I think the key thing is, we've just done a big exercise to remove ROT from the website... let's make sure we aren't adding any ROT in with new text.  
Eg. in the text below we say "some of our team members are actively sharing thoughts on Twitter". This is a bit of a duplication as directly below we have the team pictures and everyone who is on Twitter has a link to their Twitter handle.
And we mention the fact that we are blogging on this page and the home page and we have a link for 'BLOG' in the header.

Reply:
Thank you for providing comments Rachel. In my opinion,".... some of our team members are actively sharing thoughts on Twitter" is not a duplication of the Twitter icons on each profile. It is a **highlight** which would direct a user to go and check some of the team member's Twitter accounts. Providing just Twitter accounts doesn't mean that a user is actively sharing thoughts. We might want to rephrase it but I would really mention it on the ABout us page.